compiler/testData/debug/localVariables/destructuring/lambdaCustomComponentNsMultiline.kt

```diff
@@ -0,0 +1,48 @@
+// IGNORE_BACKEND: JVM_IR
+
+// FILE: test.kt
+class MyPair(val x: String, val y: String) {
+    operator fun component1(): String {
+        return "O"
+    }
+
+    operator fun component2(): String {
+        return "K"
+    }
+}
+
+fun foo(a: MyPair, block: (MyPair) -> String): String = block(a)
+
+fun box() {
+    foo(MyPair("X", "Y")) { (x, y) -> x + y }
```

Comment:
Looks like you intended to make this test multiline but forgot? Right now it's equivalent to `lambdaCustomComponentNs.kt`

Reply:
Thanks, absolutely right.